projects.js

```diff
@@ -0,0 +1,118 @@
+class RepositoryProjectsManager {
+  constructor({ owner, repository, octokit }) {
+    this.owner = owner;
+    this.repositoryName = repository;
+    this.octokit = octokit;
+    this.clientMutationId = `sync-repository-projects-${owner}-${repository}`;
+  }
+
+  async init() {
+    // the GitHub Action's event can contain the "old" GraphQL node id.
+    // this produces deprecation warnings. as a workaround, look up the "new" ID.
+    // https://github.blog/changelog/label/deprecation/
+    const { organization } = await this.octokit.graphql(
+      `query {
+         organization(login: "${this.owner}") {
+           id
+           name
+         }
+      }`,
```

Comment:
would this be better for the indentation?

```suggestion
    const { organization } = await this.octokit.graphql(`
      query {
        organization(login: "${this.owner}") {
          id
          name
        }
      }`,
```

Reply:
@jansiwy I've applied this pattern. It works fine everywhere except here because the npm run lint requires it as updated now. Please check [Extend comment on clientMutationId](https://github.com/babbel/sync-repository-projects/pull/3/commits/7a4df5e8426582e121d46a0f6bfffbcb298da25c) and let me know if this is fine for you.